doc/source/examples/index.rst

```diff
@@ -18,17 +18,20 @@ Here are a series of examples using MAPDL with ``ansys-mapdl-core``.
 
 .. === TECHNOLOGY SHOWCASE EXAMPLES ===
 
-.. toctree::
-    :hidden:
-    :maxdepth: 3
+The PyMAPDL technology showcase examples are more complex test cases than
+the ones presented in the <gallery_examples_>. They demonstrate how to use
+PyMAPDL to effectively and accurately solve interdisciplinary problems from
+a variety of industries and engineering fields.
 
-    technology_showcase_examples/techdemo-1/ex_0-tecbrakesqueal
-    technology_showcase_examples/techdemo-20/20-example-technology-showcase-dynamic-simulation-PCB
-    technology_showcase_examples/techdemo-28/ex_28-tecfricstir
+The examples initially come from the `APDL Technology Showcase Manual
+<_tech_demo_intro_>`_ and are modeled with MAPDL files. Some of them have been
+reproduced in Python files using PyMAPDL with the ``ansys-mapdl-core``
+library.
 
-.. include:: technology_showcase_examples/index.rst
-   :start-line: 2
+The examples are hosted in the  `PyMAPDL Techdemos repository <_tech_demos_repo_>`_.
 
+The documentation to the exposed examples can be found in the
+`PyMAPDL Techdemos documentation <_tech_demos_doc_>`_.
```

Comment:
I wonder if we should link each of the examples (as it was) or just link the whole documentation.

Reply:
If you go to the docs page you'll find the same table as before. I'd leave it as is, but it's up to you honestly :)

Reply:
Sure, I can do that. I can add a link for each example to make it look more like the previous version.